spec/presenters/reports/repository_presenter_spec.rb

```diff
@@ -1,9 +1,125 @@
+        :visited_without_bp_taken_with_ltfu_rates,
+        :visited_without_bp_taken,
+        :adjusted_diabetes_patient_counts_with_ltfu,
+        :adjusted_diabetes_patient_counts,
+        :bs_below_200_patients,
+        :bs_below_200_rates,
+        :bs_below_200_with_ltfu_rates,
+        :bs_below_200_breakdown_rates,
+        :bs_200_to_300_breakdown_rates,
+        :bs_over_300_breakdown_rates,
+        :bs_over_200_breakdown_rates,
+        :bs_200_to_300_patients,
+        :bs_200_to_300_rates,
+        :bs_200_to_300_with_ltfu_rates,
+        :bs_over_300_patients,
+        :bs_over_300_rates,
+        :bs_over_300_with_ltfu_rates,
+        :diabetes_missed_visits,
+        :diabetes_missed_visits_with_ltfu,
+        :diabetes_missed_visits_rates,
+        :diabetes_missed_visits_with_ltfu_rates,
+        :visited_without_bs_taken_rates,
+        :visited_without_bs_taken_with_ltfu_rates,
+        :visited_without_bs_taken,
+        :diabetes_patients_with_bs_taken,
+        :diabetes_patients_with_bs_taken_breakdown_rates,
+        :diabetes_patients_with_bs_taken_breakdown_counts,
+        :dead,
+        :diabetes_dead,
+        :under_care,
+        :diabetes_under_care,
+        :overdue_patients,
+        :overdue_patients_rates,
+        :contactable_overdue_patients,
+        :contactable_overdue_patients_rates,
+        :patients_called,
+        :patients_called_rates,
+        :contactable_patients_called,
+        :contactable_patients_called_rates,
+        :patients_called_with_result_agreed_to_visit,
+        :patients_called_with_result_remind_to_call_later,
+        :patients_called_with_result_removed_from_list,
+        :contactable_patients_called_with_result_agreed_to_visit,
+        :contactable_patients_called_with_result_remind_to_call_later,
+        :contactable_patients_called_with_result_removed_from_list,
+        :patients_called_with_result_agreed_to_visit_rates,
+        :patients_called_with_result_remind_to_call_later_rates,
+        :patients_called_with_result_removed_from_list_rates,
+        :contactable_patients_called_with_result_agreed_to_visit_rates,
+        :contactable_patients_called_with_result_remind_to_call_later_rates,
+        :contactable_patients_called_with_result_removed_from_list_rates,
+        :patients_returned_after_call,
+        :patients_returned_after_call_rates,
+        :patients_returned_with_result_agreed_to_visit_rates,
+        :patients_returned_with_result_remind_to_call_later_rates,
+        :patients_returned_with_result_removed_from_list_rates,
+        :contactable_patients_returned_after_call,
+        :contactable_patients_returned_after_call_rates,
+        :contactable_patients_returned_with_result_agreed_to_visit_rates,
+        :contactable_patients_returned_with_result_remind_to_call_later_rates,
+        :contactable_patients_returned_with_result_removed_from_list_rates
+      ]
+      expect(subject.keys).to match_array(expected_keys)
```

Comment:
**quibble (non-blocking)**: I might just inline `subject` so that I don't have to scroll up to see what it expands to.

Reply:
done